### ISSUE TITLE
What release comes after 2019.1.1?

### ISSUE THREAD

**Issue by autumnontape** [garglk/garglk#538]
Hello! I'm the maintainer of [the `gargoyle` package in the AUR](https://aur.archlinux.org/packages/gargoyle/), which tracks releases. ([`gargoyle-git`](https://aur.archlinux.org/packages/gargoyle-git/) tracks this repository.)

It was 8 years between feature releases last time, and it's been a couple years since the last one now, but the latest release already feels older than it is because:

* It depends on the deprecated GTK+ 2 and SDL 1.2 libraries
* It uses the Jam build system, which Arch's makepkg doesn't have integration for, so it's difficult to use the user's preferred level of parallelism when building
* It fails to build on current versions of GCC without either patches or options

All of these have been fixed in git, so I think it would make sense to get those changes into a release soon. Good job keeping this old codebase fresh, by the way!

**Comment by cspiegel** [comment]
I completely agree, and I've been working on a few straggling bugs and beating some stuff into shape, hoping for a release no later than the end of the year, but likely sooner.  There are no more major changes needed, so there's nothing big holding up the release.

If there's anything that'd help packaging, let me know.

**Comment by autumnontape** [comment]
All right, thank you!

**Comment by nickbe** [comment]
Is it possible to have at least some preliminary release until the signing problem is solved. (No installer - just binaries)?

**Comment by cspiegel** [comment]
Signing's not going to hold up the release, but there were three issues (TADS, HiDPI, and @save opcode updates in Bocfel) which did.  TADS was merged in a few weeks ago, I just merged the latest HiDPI changes in, and I'll have the Bocfel update in today.  That _should_ be the kickoff point for a new release.

But if you're looking for binaries, check the [Actions](https://github.com/garglk/garglk/actions) tab: there's an AppImage for Linux, an installer plus binaries for Windows, and a DMG for macOS.  The Windows stuff is under MinGW (I really should rename that ...)
